inventory-manager.lic

```diff
@@ -0,0 +1,125 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#invdb
+=end
+
+class InventoryManager
+  include DRC
+  
+  def initialize
+    @active_character = Char.name.to_sym
+    @item_db_path = "data/inventory.yaml"
+
+    arg_definitions = [
+			[
+        { name: 'save', regex: /save/i, description: "Save new or update current character's items to the database." }
+			],
+      [
+        { name: 'search', regex: /search/i, description: 'Start a search across all characters for specified item.' },
+				{ name: 'item', regex: /\w+/i, optional: false, description: 'Item to find.' }
+			],
+			[
+				{ name: 'list', regex: /list/i, description: "Print out a list of names in the database." },
+        { name: 'name', regex: /\w+/i, optional: true, description: "List full inventory for specified character. (Can spam!)" }
+			],
+			[
+				{ name: 'remove', regex: /remove/i, description: "Remove character's inventory." },
+				{ name: 'name', regex: /\w+/i, optional: false, description: 'Character to remove.' }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    pause
```

Comment:
💡 Why `pause` here?

Reply:
I am not sure, I've seen other scripts do this within dependency so I added it. Figured it was some type of "thing".